cdc/sinkv2/ddlsink/ddl_sink.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ddlsink
+
+import (
+	"context"
+
+	"github.com/pingcap/tiflow/cdc/model"
+)
+
+// DDLEventSink is the interface for sink of DDL events.
+type DDLEventSink interface {
```

Comment:
How about renaming to `OwnerSink`? In fact `WriteCheckpointTs` isn't relative with DDL.

Reply:
OwnerSink may be a bit strange. Because it is even less obvious from the name what its duties are. So I prefer to keep it.